relate/urls.py

```diff
@@ -395,6 +395,14 @@
         "/$",
         course.flow.view_flow_page,
         name="relate-view_flow_page"),
+    re_path(r"^course"
+        "/" + COURSE_ID_REGEX
+        + "/flow-session-tabbed"
+        "/(?P<flow_session_id>[0-9]+)"
+        "/(?P<page_ordinal>[0-9]+)"
+        "/$",
+        course.flow.view_flow_tabbed_page,
+        name="relate-view_flow_tabbed_page"),
```

Comment:
It occurred to me that I can't easily link to this from a flow page. If you changed the route to `/flow-session/1234/0/ext-resource-tabs`, then `[Open tabs](ext-resource-tabs)` should do the trick. (We should test this before merging.)